### PR TITLE
Vanila nerf (#1)

### PR DESCRIPTION
**Pull request by teshaTe**
* [ADDED] data loader

* [ADDED] conda environment file

* [ADDED] empty class for nerf meshing

* [ADDED] empty classes with vanila NeRF models

* [ADDED] empty class with rendering functions

* [ADDED] empty tool for meshing NeRF

* [ADDED] empty tool for training NeRF

* [ADDED] empty file for futire adding of coordinate transforms between various frames of ref.

* [UPDATED] Added rendering functionality for rendering the NeRF model + comments

* [UPDATED] fix the typo in the name of the file

* [UPDATED] number of bins now part of the constructor init

* [ADDED] two files that initiate the folders as modules

* [UPDATED] moved to a new folder

* [UPDATED] moved to a new folder

* [UPDATED] renamed the file + added implementations of the models

* [UPDATED] start assembling of the training script

* [UPDATED] finished training the vanilla nerf tool

* [UPDATED] documentation

* [UPDATED] training tool

* [UPDATED] conversion to torch type + adding type conversion

* [ADDED] tests for data loader and data for testing

* [UPDATED] added pytest as a dependence

* [UPDATED] float division

* [UPDATED] fixed problem with img normalization

* [REMOVED] the file

* [UPDATED] added extra checks

* [UPDATED] added printing output

* [UPDATED] generate rays -> revert all computations to numpy

* [UPDATED] added meshing implementation for the NeRF

* [UPDATED] added meshing implementation for the NeRF (tool)

* [UPDATED] project description

### CODE REVIEW

**Comment by teshaTe** [comment]
Merging a working version of the Vanilla NeRF Implementation
